src/features/auth/register-form/RegisterForm.tsx

```diff
@@ -2,79 +2,113 @@ import { FunctionComponent } from "react";
+    const {name, email, password, confirmPassword, onChangeName, onChangeEmail, onChangePassword, onChangeConfirmPassword} = useRegisterFormState();
+    
+
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+    const FormBoxStyles = {
+        width: isSmallScreen ? '85%' : '350px',
+        height: '550px',
+        margin: 'auto', 
+        padding: isSmallScreen ? '18px' : '40px',
+        backgroundColor:'white',
+        display: 'flex',
+        flexDirection:'column',
+        alignItemns:'center',
+        justifyContent:'center',
+        borderRadius:'8px',
+        marginBottom:'16px',
+    };
+
+    return <>
+        <Box sx={FormBoxStyles}>
+            <Box sx={ImageBoxStyles}>
+            <img src={ucabLogo} alt="UCAB Logo" style={UcabLogoStyles} />
+            </Box>
+            <Typography sx={TextFieldTypography}>Nombre</Typography>
+            <TextField
+                value={name}
+                label=""
+                type="name"
+                onChange={onChangeName}
+                disabled={disabled}
+                sx={TextFieldStyles}
+                InputProps={{ sx: { borderRadius: '11px' } }}
+
+            />
+            <Typography sx={TextFieldTypography}>Correo electronico</Typography>
+            <TextField
+                value={email}
+                label=""
+                type="email"
+                onChange={onChangeEmail}
+                disabled={disabled}
+                sx={TextFieldStyles}
+                InputProps={{ sx: { borderRadius: '11px' } }}
+
+            />
+            <Typography sx={TextFieldTypography}>Contraseña</Typography>
+            <TextField
+                value={password}
+                label=""
+                type="password"
+                onChange={onChangePassword}
+                disabled={disabled}
+                sx={TextFieldStyles}
+                InputProps={{ sx: { borderRadius: '11px' } }}
+
+            />
+            <Typography sx={TextFieldTypography}>Confirmar contraseña</Typography>
+            <TextField
+                value={confirmPassword}
+                label=""
+                type="password"
+                onChange={onChangeConfirmPassword}
+                disabled={disabled}
+                sx={{...TextFieldStyles, marginBottom: '26px'}}
+                InputProps={{ sx: { borderRadius: '11px' } }}
+
+            />
+            <Button
+                variant="contained"
+                color="primary"
+                type="submit"
+                disabled={disabled}
+                onClick={()=>{
+                    onSubmit(name, email, password, confirmPassword);    
+                }}
+                sx={ButtonStyles}
+            >
+            Registrarse
+            </Button>
+            <Box sx={UnderFormStyles}> 
+                <Box sx={{display:'inline-block',}}>¿Ya tienes cuenta?</Box>   &nbsp;  &nbsp;  &nbsp;
```

Comment:
Misma sugerencia que en el login
```suggestion
                <Box sx={{display:'inline-block',}}>¿Ya tienes cuenta?</Box>
```